NetCoreRestApi/ServiceLayer/Converters/CategoryServiceConverter.cs

```diff
@@ -1,78 +1,27 @@
-﻿using DataLayer.Models;
-using Common.Converter;
+﻿using Common.Converters;
+using DataLayer.Models;
 using ServiceLayer.DataTransferObjects;
-using System.Collections.Generic;
+using System;
+using System.Linq.Expressions;
 
 namespace ServiceLayer.Converters
 {
-    public class CategoryServiceConverter : IConverter<CategoryDTO, CategoryModel>
+    public class CategoryServiceConverter<TId> : BaseConverter<CategoryDto<TId>, CategoryModel<TId>>
```

Comment:
Fix to class CategoryServiceConverter : BaseConverter<CategoryDto, CategoryModel>

Reply:
done